tasks/src/mem_db.rs

```diff
@@ -235,6 +235,23 @@ impl InMemoryTaskDb {
         hex::decode(proof)
             .map_err(|_| TaskManagerError::SqlError("couldn't decode from hex".to_owned()))
     }
+
+    fn prune_aggregation(&mut self) -> TaskManagerResult<()> {
+        self.aggregation_tasks_queue.clear();
```

Comment:
The API was meant just for pruning existing statuses, not cancelling in-flight proofs. Currently we would need to cancel before pruning.
We could implement listing out all currently running proofs and cancel them, then prune. This would have to be implemented for both regular and aggregation proofs.